examples/foomedical/src/pages/health-record/Responses.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box, Stack, Text, Title, useMantineTheme } from '@mantine/core';
+import { formatDateTime, getReferenceString } from '@medplum/core';
+import { Patient } from '@medplum/fhirtypes';
+import { useMedplum, useMedplumProfile } from '@medplum/react';
+import { IconChevronRight } from '@tabler/icons-react';
+import { useNavigate } from 'react-router-dom';
+import { InfoButton } from '../../components/InfoButton';
+import { InfoSection } from '../../components/InfoSection';
+
+export function Responses(): JSX.Element {
+  const medplum = useMedplum();
+  const theme = useMantineTheme();
+  const navigate = useNavigate();
+  const profile = useMedplumProfile() as Patient;
+  const responses = medplum
+    .searchResources('QuestionnaireResponse', `source=${getReferenceString(profile)}&_sort=-authored`)
```

Comment:
Same as in the other cases, making use of suspense for displaying the loading spinner